repository.rosinstall

```diff
@@ -21,20 +21,19 @@
 - git:
     local-name: universal_robot
     uri: 'https://github.com/ros-industrial/universal_robot.git'
-    version: kinetic-devel
+    version: 'kinetic-devel'
```

Comment:
I think that our default format is without single quote.
We should remove it from the other lines instead.